cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -28,16 +26,15 @@ var (
 )
 
 type server struct {
-	config  *Config
-	logger  *zap.Logger
-	server  *queryApp.Server
-	jtracer *jtracer.JTracer
+	config *Config
+	server *queryApp.Server
+	telset component.TelemetrySettings
 }
 
 func newServer(config *Config, otel component.TelemetrySettings) *server {
 	return &server{
 		config: config,
-		logger: otel.Logger,
+		telset: otel,
```

Comment:
do you actually need the original `component.TelemetrySettings` anywhere? Why not convert to our own right here?